torch/csrc/jit/codegen/cuda/test/test_gpu_validator.h

```diff
@@ -36,6 +37,10 @@ class NVFuserTest : public ::testing::Test {
       GTEST_SKIP() << "skipping tests on pre-PASCAL GPUs";
     }
   }
+
+  void TearDown() override {
+    c10::cuda::CUDACachingAllocator::emptyCache();
```

Comment:
Why do we need to do this every time a test is done? Does it involve cudaFree? If so, wouldn't this running the tests slower?